android/titanium/src/java/org/appcelerator/titanium/io/TiFile.java

```diff
@@ -95,6 +97,71 @@ public boolean isWriteable()
 		return file.canWrite();
 	}
 
+	@Override
```

Comment:
If you're going that way then it would just be better to edit TiFileProxy instead because less changes would need to be made.